src/core/transform.js

```diff
@@ -24,13 +24,41 @@ import p5 from './main';
  * > <img style="max-width: 150px" src="assets/transformation-matrix.png"
  * alt="The transformation matrix used when applyMatrix is called"/>
  *
+ * <img style="max-width: 150px" src="assets/transformation-matrix-4-4.png"
```

Comment:
150px width is a bit too small for this matrix. You can preview this using `grunt yui:dev`.